src/FormBundle/Mailer/Mailer.php

```diff
@@ -53,8 +53,9 @@ public function __construct(TranslatorInterface $translator, RequestStack $reque
     public function sendNotificationMail(FormInterface $form, PostInterface $post)
     {
         $body = $this->templating->render('OpiferFormBundle:Email:notification.html.twig', ['post' => $post]);
+        $subject = $form->getName() . ' (post id:' . $post->getId() . ')';
```

Comment:
Shortened the prefix and added a space between the last character and the post id itself.

```suggestion
        $subject = $form->getName() . ' (post ' . $post->getId() . ')';
```